frontend/src/css/App.css

```diff
@@ -1,3 +1,17 @@
 .app {
   text-align: center;
 }
+
+.navbar {
+  padding-bottom: 15px !important;
```

Comment:
Yes they are necessary, since I needed to override the Aurora classes in order to customize the header thickness and the bottom tab border in order to keep the tab selected. If I'm not putting that 'important' parameter, these css custom classes are not even considered.

I used the most possible the Aurora classes, meaning that I put the least customization possible as well.

Is there other ways to do that? Maybe, but I'm not sure how. If you have other ideas, I can try them.

Reply:
@caleybrock, I found another way to make that working without these **!important** parameters.
See commit #https://github.com/code-for-canada/project-thundercat/pull/70/commits/f9664c347581a5a4bd92a830b1a7f9660e678a83 and commit #https://github.com/code-for-canada/project-thundercat/pull/70/commits/31b7c9c67cc7bdd0b6e8da055f8ab172d0644546